src/applications/coronavirus-screener/components/FormResult.jsx

```diff
@@ -43,17 +46,27 @@ export default function FormResult({ formState }) {
   );
 
   function recordScreeningToolEvent(screeningToolResult) {
-    if (!resultSubmitted) {
+    if (!resultSubmitted.isSubmitted) {
+      const timeToComplete = moment.duration(
+        moment().diff(resultSubmitted.startTime, 'seconds'),
+      );
+      // TODO: format timeToComplete to hh:mm:ss
```

Comment:
I would avoid adding another dependency for only this code, since we are planning to deprecate Moment at some point. Play around with this and see if it works for what you need.

Note that `moment.duration` expects the duration to be in milliseconds, so I updated that accordingly.

```
const timeToComplete = moment.duration(
  moment().diff(resultsSubmitted.start, 'milliseconds'),
);

const formattedTimeToComplete = [
  timeToComplete.hours(),
  timeToComplete.minutes(),
  timeToComplete.seconds(),
].map(segment => segment.toString.padStart(2, '0')).join(':');
```

Reply:
Thanks, this works well!